LSP.sublime-settings

```diff
@@ -8,6 +8,22 @@
   // Valid values are "never", "always" and "saved"
   "auto_show_diagnostics_panel": "saved",
 
+  // A dictionary of code action identifiers that should be triggered on save.
+  //
+  // Code action identifiers are not officially standardized so refer to specific
+  // server's documentation on what is supported but `source.fixAll` is commonly
+  // used to apply fix-on-save code action.
+  //
+  // It's also supported to set this in project-specific settings in which case
+  // it will only apply in that specific project. Setting it here will apply globally,
+  // unless overridden by project-specific settings.
+  //
+  // Only "source.*" actions are supported.
+  "lsp_code_actions_on_save": {},
```

Comment:
Instead of just having an empty object for `lsp_code_actions_on_save`,
we could give a hint of how that object looks:
```
  "lsp_code_actions_on_save": {
    // "source.fixAll": true
  },
```